docs/sematic-vs.md

```diff
@@ -56,35 +56,97 @@ But bear with us, we'll catch up fast!
   master. Sematic tries to provide working solutions out-of-the-box so that you
   can focus on your expertise instead of infrastructure.
 
-* **Dynamic graph** – In KFP, DAGs are fixed ahead of time. Sematic lets up loop over configurations, run different branches of your pipelines depending on the outcome of upstream steps, and even do hyperparameter tuning.
+* **Dynamic graph** – In KFP, DAGs are fixed ahead of time. While it does provide
+[some mechanisms](https://kubeflow-pipelines.readthedocs.io/en/stable/source/kfp.dsl.html#kfp.dsl.Condition)
+to modify pipeline behavior mid-execution, you are limited by the provided control flow
+statements. Sematic lets you use standard python control flow to loop over configurations,
+run different branches of your pipelines depending on the outcome of upstream steps,
+and even do hyperparameter tuning.
 
-* **Lineage tracking** – KFP does not keep a rigorous track of all assets and artifacts flowing between your steps. You need to implement that yourself on top of it. In Sematic, lineage tracking is a first-class citizen. All inputs and outputs of all Sematic Functions are tracked and versioned.
+* **Lineage tracking** – KFP has some basic tracking of artifacts flowing between your steps, but is limited in the
+type of data that can be tracked in this way. If you want to have tracking for high-level, typed objects like "models"
+instead of just paths to blobs, you need to implement that yourself on top of it. In Sematic, lineage tracking of richly
+typed artifacts is a first-class citizen. All inputs and outputs of all Sematic Functions are tracked and versioned.
 
 * **Semantic UI** – The KFP UI offers barebones tools to monitor and inspect runs, but no rich visualization or advanced features.
 
-### MLFlow
-
-### Dagster
+### ... MLFlow Pipelines
+
+* **Flexibility** - To use
+[MLFlow pipelines](https://www.mlflow.org/docs/latest/pipelines.html#pipeline-templates),
+you are limited to a pre-defined set of pipeline templates. Sematic lets you build any
+pipeline structure you like.
+
+* **Pythonic SDK** - Using MLFlow requires using a yaml-based configuration combined with
+Jinja templating to enable hierachical config specifications. Sematic lets you stick with
+simple python for constructing your pipeline and feeding in configurations and other inputs.
+
+* **Powerful compute access** - MLFlow pipelines run natively on Databricks VMs, or
+you can execute them on your own VMs if you manually integrate it with your own job
+execution solution. Sematic lets each step in your pipeline run with the computing
+resources that are appropriate for it.
+
+### ... Dagster
+* **Semantic UI** – Sematic has rich visualizations for the inputs and outputs at all levels
+of your pipelines. Instead of specifying configurations with yaml in your browser, you can
+fill out web forms that have just the fields expected, with appropriate validation. You can
```

Comment:
We don't have this functionality yet though. Should we say "you will soon be able to..." with maybe a link to a Trello ticket on the public roadmap?

Reply:
added a `(this feature coming soon!)`